src/brc/api/tests/test_dso_api_strategy.py

```diff
@@ -62,7 +62,8 @@ def assertErrorResponse(self, view, expected_data: dict):
 
         exc_class = view.exception.__class__.__name__
         expected_data["type"] = f"http://testserver/ref/fouten/{exc_class}/"
-        self.assertEqual(response.data, expected_data)
+
+        self.assertEqual(list(response.data).sort(), list(expected_data).sort())
```

Comment:
Volgens mij is dit niet de bedoeling, waarom wordt er hier gecast naar een `list`?